src/js/player.js

```diff
@@ -2416,6 +2417,12 @@ class Player extends Component {
       if (seconds < 0) {
         seconds = 0;
       }
+      if (!this.isReady_ || this.changingSrc_ || !this.tech_ || !this.tech_.isReady_ && seconds > 0) {
```

Comment:
We may want to not do a check for seconds because in a live stream, if you were to seek to zero, we'd want to then start at the beginning of the DVR window rather than at the live window.

Reply:
got it, I missed out that scenario...thanks for notice that!